### PR TITLE
330: Conditional Voting

### PR DESCRIPTION
**Pull request by pimotte**
This is a) fun and b) a set-up for declaring dependencies between rule changes/pull request.

ノミクをやれば、遊びます。If I play Nomic, I enjoy myself.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -204,6 +204,25 @@ If the implicit vote is not overwritten with an explicit vote, the validity of t
...
+In this format, items in between parentheses are optional. `[condition]` should
+be replaced with a human readable condition. If this condition cannot be resolved
+at a time when a player attempts to do so, for example because of improper formulation or
+a logical paradox, it will be resolved as being false.
+
+`[voting post]` should be replaced by the contents of a comment representing the vote,
+or an empty string. Conditional votes are a comment representing a vote.
+
+In order to perform an action based on resolution of one of more conditional votes,
+a player must post a comment describing the resolution before performing said action.
```
Comment (MrHug) [inline README.md:298]: Hmm I like that, but what if there is a misinterpretation. I.e. something is considered true by player A, but false by a player B. Do we invoke judgement, do we put A on Trial, what happens?

Comment (pimotte) [inline README.md:298]: If two players consider it to be different results, then the condition cannot be resolved, and it should be false. So if A were to merge a PR, for which the truthiness is required, then, yes, they might be put on trial.

**Comment by pimotte** [comment]
More opinions?

**Comment by MrHug** [comment]
Well I am generally in favour, maybe just to prevent Trials/Judgements from happening all too quickly, we could add a minimum duration between resolving a condition and performing the action. This duration can be nullfiied if the original author of the conditional vote confirms the resolution?

Also what happens if I post a conditional vote and then a real one later? I'm guessing the conditional vote just gets cancelled?

**Comment by pimotte** [comment]
Yes, at least that's how we've treated :-1: then :+1: or vice versa 

With regards to duration, I'm ambivalent about that. I'd like to see how things go with the current rule, and if it indeed leads to too many Trials/Judgments, we can adapt. I think it won't be so bad.

**Comment by MrHug** [comment]
Alright, well I like Trials just as much as the next guy, so sure ;)

And I can live with how we treat them, so vote up it is.

**Comment by MrHug** [comment]
:+1:

**Comment by jdonkervliet** [comment]
:+1:

**Comment by pimotte** [comment]
3/4 votes, so merge.

**Pull request merged**